Kconfig

```diff
@@ -39,4 +39,10 @@ config OPTEE_TEE_SUPPLICANT
 	  applications from application and perform file system operation
 	  on non-volatile memory devices which inaccessible from OP-TEE
 
+config OPTEE_LIBSETEEC
+	bool "Enable libseteec"
+	depends on OPTEE_LIBTEEC
```

Comment:
If this code depends on CONFIG_POSIX_API, why not add this dependency here? It is most logical.

Reply:
Agree. Fixed